src/AsideBody/AsideBody.js

```diff
@@ -8,33 +7,24 @@ import WishList from '../WishList/WishList.js';
 function AsideBody(props) {
   if (props.asideView === "myCardList") {
     return (
-      <div>
-        <CardList 
-          cardNames={props.cardNames} 
-          removeListItem={props.removeListItem}
-          setCardCount={props.setCardCount}
-          userCardsData={props.userCardsData} 
-          compareBuilds={props.compareBuilds}/>
-      </div>
+      <CardList 
```

Comment:
Because of these changes, I think that AsideBody can be removed altogether and its job can be done by Aside.